CMakeLists.txt

```diff
@@ -98,6 +100,12 @@ target_link_libraries(alicia-server PRIVATE
 target_include_directories(alicia-server PUBLIC
         "${PROJECT_BINARY_DIR}/generated")
 
+# Allow RanchDirector.cpp to exceed the default section limit
```

Comment:
Please do this instead,
https://github.com/Story-Of-Alicia/alicia-server/blob/feature/shop/CMakeLists.txt#L35